vault/logical_system_paths.go

```diff
@@ -584,6 +584,7 @@ func (b *SystemBackend) configPaths() []*framework.Path {
 				logical.UpdateOperation: &framework.PathOperation{
 					DisplayAttrs: &framework.DisplayAttributes{
 						OperationVerb: "initialize",
```

Comment:
Just a note for others, but the subsequent place this is done (line 817 originally prior to this change) isn't affected as the entire group of operations has `OperationPrefix: "rekey-attempt"`, making the full function call `rekey_attempt_initialize(...)` in the Ruby client. 

Further this is the [only keyword](https://stackoverflow.com/questions/16099073/ruby-reserved-class-method-names) that appears to be reserved:

```
$ comm -12 /tmp/ruby-reserved-keywords /tmp/operations 
initialize
```